gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```
@@ -752,6 +752,69 @@ paths:
                 default:
                     $ref: "#/components/responses/Error"
 
+    /environments/{envId}/apis/{apiId}/subscriptions:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+            - $ref: "#/components/parameters/apiIdParam"
+        get:
+            parameters:
+                - name: applicationIds
+                  description: List of application ids to filter on.
+                  in: query
+                  explode: false
+                  schema:
+                      type: array
+                      items:
+                          type: string
+                - name: planIds
+                  description: List of plan ids to filter on.
+                  in: query
+                  explode: false
+                  schema:
+                      type: array
+                      items:
+                          type: string
+                - name: statuses
+                  in: query
+                  required: false
+                  description: List of status filters.
+                  explode: false
+                  schema:
+                      type: array
+                      items:
+                          $ref: "#/components/schemas/SubscriptionStatus"
+                - name: apiKey
+                  in: query
+                  description: Api key associated to the subscription to filter on.
+                  schema:
+                      type: string
+                - name: expands
+                  in: query
+                  description: Expansion of data to return in subscriptions.
+                  schema:
+                      type: array
+                      items:
+                          type: string
+                          enum:
+                              - application
+                              - plan
+                              - apiKeys
+                  explode: false
+                - $ref: "#/components/parameters/pageParam"
+                - $ref: "#/components/parameters/perPageParam"
+            tags:
+                - API Subscriptions
+            summary: Get the API's subscriptions
+            description: |
+                List subscriptions for a given API, always filtered by a `status` and the possibility to filter by `security`.
+                User must have API_PLAN[READ] or API_LOG[READ] permissions to access endpoint.
```

Just a suggestion, because it will have an impact on the way the documentation is displayed.
```suggestion
                List subscriptions for a given API, always filtered by a `status` and the possibility to filter by `security`.

                User must have API_PLAN[READ] or API_LOG[READ] permissions to access endpoint.
```